glean-core/rlb/Cargo.toml

```diff
@@ -35,6 +43,7 @@ serde = { version = "1.0.104", features = ["derive"] }
 uuid = { version = "0.8.1", features = ["v4"] }
 chrono = { version = "0.4.10", features = ["serde"] }
 time = "0.1.40"
+libloading = "0.6.6"
```

Comment:
Do we need `libloading` if not `glean-dynamic`?

Reply:
Nope. missed that.